src/iter.rs

```diff
@@ -75,22 +74,6 @@ impl<K: Eq + Hash, V, S: BuildHasher + Clone> Iterator for OwningIter<K, V, S> {
     }
 }
 
-unsafe impl<K, V, S> Send for OwningIter<K, V, S>
-where
-    K: Eq + Hash + Send,
-    V: Send,
-    S: BuildHasher + Clone + Send,
```

Comment:
This is quite inconsistent with other `Send/Sync` implementations, they all require `K: Send, V: Send, S: Clone`.

Once specified which version is correct (`S: Send` or `S: Clone`) we should unify all `Send` (and `Sync`) implementations.

Reply:
Also, it would be wise to add `Safety` comments to Send (Sync) implementations, possibly including rationale why we use `S: Clone` in place of `S:Send`

Reply:
Is there still a willingness accept this type of change? I am happy to go through the existing code base and relax the  trait bounds where possible. I'll leave Send/Sync bounds where they are.